recipe/meta.yaml

```diff
@@ -10,19 +10,19 @@ source:
   sha256: 2b91300f4f1926b396c2c90185844eb1a3d5ec39ea6138832d119da0a208f4d9
 
 build:
-  number: 0
+  number: 1
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
     - pip
-    - python
+    - python >=3.5
     - setuptools
   run:
     - flake8 >=3.2.1,<4
-    - isort >=4.3.5,<5
-    - python
+    - isort >=4.3.5,<6
```

Comment:
Upstream, this seems to have been relaxed: https://github.com/gforcada/flake8-isort/blob/4.0.0/setup.py#L60

This constraint is causing difficulty installing flake8-isort with python 3.9 because only newer versions of isort have been built with that python version.